tests/vmi_gpu_test.go

```diff
@@ -67,6 +68,7 @@ var _ = Describe("[Serial][sig-compute]GPU", Serial, func() {
 
 	Context("with ephemeral disk", func() {
 		It("[test_id:4607]Should create a valid VMI but pod should not go to running state", func() {
+			var pod *k8sv1.Pod
```

Comment:
Please declare it as close as possible to the usage.

Reply:
Done